backend/controller/user.go

```diff
@@ -26,6 +26,24 @@ func GetUser(w http.ResponseWriter, r *http.Request) {
 	helpers.JsonWriter(w, user)
 }
 
+func GetUserByEmail(w http.ResponseWriter, r *http.Request) {
```

Comment:
this isn't necessary you can use the getFirstTable function in the helper function as long as you pass the param

Reply:
? not following, i believe this function is necessary because we have no other endpoint to get a user by email

Reply:
Get first table takes a param, if you look at getby user id or pk or whatever we pass the userId but you can just as easily pass email instead and it functions the same

Reply:
I can put sample function when i get home

Reply:
That's what im doing in this function so im confused what you mean